resources/serverless/profile-production.yaml

```diff
@@ -16,4 +16,9 @@ webhook:
 docker-registry:
   resources:
     requests:
-      cpu: 50m
\ No newline at end of file
+      cpu: 50m
+
+containers:
+  manager:
+    envs:
+      functionBuildMaxSimultaneousJobs: "5"
```

Comment:
I wonder if we wouldn't be able to handle more than 5 simultaneous builds in production configuration.

Reply:
Indeed maybe we should revisit the acceptance criteria of the task, let's discuss

Reply:
As discussed, for now, we'll go with 5